src/main/scala/com/cognite/sdk/scala/common/dataTypes.scala

```diff
@@ -21,18 +21,23 @@ final case class ItemsWithRecursiveAndIgnoreUnknownIds(
 final case class SdkException(
     message: String,
     uri: Option[Uri] = None,
-    requestId: Option[String] = None
+    requestId: Option[String] = None,
+    responseCode: Option[Int] = None
 ) extends Throwable(message) {
   override def toString: String = {
-    val uriMessage = uri.map(u => s", in response to request sent to ${u.toString()}").getOrElse("")
-    val requestIdMessage = requestId.map(id => s", with request id ${id}").getOrElse("")
+    val responseCodeMessage =
+      responseCode.map(c => s" (with response code ${c.toString})").getOrElse("")
+    val uriMessage = uri
+      .map(u => s", in response$responseCodeMessage to request sent to ${u.toString()}")
+      .getOrElse("")
+    val requestIdMessage = requestId.map(id => s", with request id $id").getOrElse("")
     val superString: String = super.toString
```

Comment:
We normally don't have type annotation on vals like this, so I guess we could avoid it here as well.

Reply:
I agree, but I didn't do it ;( I can use this opportunity to fix it, though!